easybuild/easyconfigs/r/R/R-4.1.2-foss-2021b.eb

```diff
@@ -3215,6 +3215,24 @@ exts_list = [
     ('catlearn', '0.8', {
         'checksums': ['0d13da431a5511b025090b7ffaa3765b94b33c7de5a7cc1cec8c50437755646e'],
     }),
+    ('EValue', version, {
+        'checksums': ['d9f3feb3da2efaf4e5d794e05475be62666192a7c855e373c3d369324078690c'],
+    }),
+    ('ggdag', '0.2.4', {
+        'checksums': ['372ec4ef0ded9637304a19b50bce908e84a034e54dbd2a52a061bb747b2b08a0'],
+    }),
+    ('dagitty', '0.3-1', {
+        'checksums': ['7d44b5d259ec3fef776a7e3fcb21d1c379f930d6ae9ae5fbfff54494ad78e8a8'],
+    }),
+    ('metadat', '1.0-0', {
+        'checksums': ['ed335a4eb2145fffde20e36077e360f9d36a615e9d93453d2bbd6f5967db4b9c'],
+    }),
+    ('MetaUtility', '2.1.2', {
```

Comment:
@deniskristak This one should be moved above `EValue`, otherwise installation of `EValue` fails with:

```
ERROR: dependency MetaUtility is not available for package EValue
```

See also `Imports` at https://cran.r-project.org/web/packages/EValue/

Currently the extensions in `exts_list` must be in the right order, to ensure that installation works when `--parallel-extensions-install --experimental` is *not* being used...

Reply:
done